README.md

```diff
@@ -9,7 +9,7 @@ To easily see what LXD is about, you can [try it online](https://linuxcontainers
 ## CI status
 
  * Travis: [![Build Status](https://travis-ci.org/lxc/lxd.svg?branch=master)](https://travis-ci.org/lxc/lxd)
- * Jenkins: [![Build Status](https://jenkins.linuxcontainers.org/job/lxd-github-commit/badge/icon)](https://jenkins.linuxcontainers.org/job/lxd-trigger-github/)
+ * Jenkins: [![Build Status](https://jenkins.linuxcontainers.org/job/lxd-github-commit/badge/icon)](https://jenkins.linuxcontainers.org/view/LXD/view/LXD%20builds/job/lxd-github-commit/)
```

Comment:
https://jenkins.linuxcontainers.org/job/lxd-github-commit/ would be better I think